conan/internal/api/install/generators.py

```diff
@@ -2,12 +2,14 @@
 import os
 import traceback
 import importlib
+import textwrap
 
 from conan.internal.cache.home_paths import HomePaths
 from conan.internal.subsystems import deduce_subsystem, subsystem_path
 from conan.internal.errors import conanfile_exception_formatter
 from conan.errors import ConanException
 from conan.internal.util.files import save, mkdir, chdir
+from .templates import ps_virtualenv_global_template, ps_virtualenv_call_scripts, sh_virtualenv_global_template, sh_virtualenv_call_scripts
```

Comment:
We practically don't use relative imports

Reply:
Oh, already changed, thanks!